apps/ui/src/helpers/quorum.ts

```diff
@@ -0,0 +1,30 @@
+import { Proposal } from '@/types';
+
+const REJECTION_QUORUM_CHOICE_INDEX = 1;
+
+export function quorumProgress(proposal: Proposal): number {
+  const totalScore =
+    proposal.quorum_type === 'rejection'
+      ? proposal.scores
+          .filter((c, i) => i === REJECTION_QUORUM_CHOICE_INDEX)
+          .reduce((a, b) => a + b, 0)
```

Comment:
Would it be the same? Looks a bit overcomplex by doing it on arrays with filter/reduce.

```suggestion
      ? proposal.scores[REJECTION_QUORUM_CHOICE_INDEX] ?? 0
```